oshmem/mca/memheap/base/memheap_base_frame.c

```diff
@@ -61,11 +61,12 @@ static int mca_memheap_base_register(mca_base_register_flag_t flags)
                           &mca_memheap_base_config.device_nic_mem_seg_size);
 
     mca_base_var_register("oshmem", "memheap", "base", "max_segments",
-                          "Maximum number of shared data segments per process",
+                          "Display a warning if the number of segments of "
+                          "shared heap exceeds this value",
```

Comment:
minor:
Display a warning if the number of segments of **the shared memheap** exceeds this value"

Reply:
re-worded